adafruit_rsa/key.py

```diff
@@ -735,28 +741,31 @@ def gen_keys(nbits, getprime_func, accurate=True, exponent=DEFAULT_EXPONENT):
 
 
 def newkeys(
-    nbits, accurate=True, poolsize=1, exponent=DEFAULT_EXPONENT, log_level="INFO"
+    nbits: int,
+    accurate: bool = True,
+    poolsize: int = 1,
+    exponent: int = DEFAULT_EXPONENT,
+    log_level: str = "INFO",
 ):
```

Comment:
I think we're missing the return type here right? 

Looks like maybe `Tuple["PublicKey", "PrivateKey"]`

Reply:
Fixed!